src/components/Avatar/Avatar.stories.tsx

```
@@ -33,8 +22,7 @@ export const Image = () => (
 );
 
 export const Initials = () => {
-  const classes = useStyles();
-  return <GenericAvatar children="JS" className={classes.color} />;
+  return <GenericAvatar children="JS" sx={{ backgroundColor: '#467F3B' }} />;
```

Biome (2.1.2)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)